examples/glasbey-colormap.py

```diff
@@ -0,0 +1,81 @@
+"""
+Glasbey colormap
+================
+
+A long-requested feature [1]_ for napari was to display labels/segmentation
+layers using a well-known discrete colormap such as Glasbey [2]_.
+
+In this example, we demonstrate displaying segmentations using custom colormaps
+with the help of the glasbey Python library [3]_. We display a segmentation
```

Comment:
Is it worth explicitly saying the user will need to install it?

Reply:
imho a gallery example is the wrong place, but I could do if people feel strongly